local-storage/src/provider.rs

```diff
@@ -70,10 +70,6 @@ impl SqliteStorageProvider {
             "CREATE INDEX IF NOT EXISTS links_root_cid on links(root_cid)",
             (),
         )?;
-        self.conn.execute(
-            "CREATE UNIQUE INDEX IF NOT EXISTS links_block_cid on links(block_cid)",
```

Comment:
Yeah, it's not necessarily unique.

Worthy of note - this unique index was so widely-specified you'd not be allowed to have that block in 2 different DAGs, much less twice in the same one.